float32 roll
float32 pitch
float32 yaw_rate
float32 thrust
---
bool success
string message
